app/models/grouped_issue.rb

```diff
@@ -8,8 +9,18 @@ class GroupedIssue < ActiveRecord::Base
   enumerize :level, in: [:debug, :error, :fatal, :info, :warning], default: :error
   # enumerize :issue_logger, in: { javascript: 1, php: 2 }, default: :javascript
   enumerize :status, in: { muted: 1, resolved: 2, unresolved: 3 }, default: :unresolved, predicates: true, scope: true
-
+  friendly_id :uuid, use: [:finders]
   before_save :check_fields
+  before_save :generate_uuid
+
+  def generate_uuid
+    if self.uuid.nil?
+      self.uuid = loop do
+        uuid = SecureRandom.hex(4)
+        break uuid unless self.class.exists?(uuid: uuid)
+      end
+    end
+  end
```

Comment:
It generates an unique id based on the column you assign it to. Tried to assign it to the message column but the url ends up looking bad.